src/monster.cpp

```diff
@@ -539,6 +540,10 @@ int monster::print_info( const catacurses::window &w, int vStart, int vLines, in
         mvwprintz( w, point( column, ++vStart ), c_yellow, _( "Aware of your presence!" ) );
     }
 
+    if( !inv.empty() ) {
+        mvwprintz( w, point( column, ++vStart ), c_white, _( "It is carrying something!" ) );
```

Comment:
```suggestion
        mvwprintz( w, point( column, ++vStart ), c_white, _( "It is carrying something." ) );
```

Just nitpicking, but I think a period is better than an exclamation mark here -- the monster is not necessarily stealing/carrying something the player cares about.

Reply:
Makes sense!